helmfile.d/charts/gatekeeper/constraints/templates/restrict-pod-disruption-budgets/constraint.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.restrictPodDisruptionBudgets.enable -}}
+apiVersion: constraints.gatekeeper.sh/v1alpha1
+kind: K8sRestrictPodDisruptionBudgets
+metadata:
+  name: elastisys-restrict-pod-disruption-budgets
+spec:
+  enforcementAction: {{ .Values.restrictPodDisruptionBudgets.enforcementAction }}
+  match:
+    kinds:
+      - apiGroups: [""]
+        kinds: ["Pod", "ReplicationController"]
+      - apiGroups: ["apps"]
+        kinds: ["Deployment", "StatefulSet", "DaemonSet", "ReplicaSet"]
+      - apiGroups: ["extensions"]
+        kinds: ["Deployment", "StatefulSet", "DaemonSet", "ReplicaSet"]
+      - apiGroups: ["batch"]
+        kinds: ["Job", "CronJob"]
```

Comment:
These do nothing at the moment right? Same with daemonset

Reply:
Yeah, this was just something I forgot to remove when I reduced the scope of the task.
Removed now.